components/inputs/input-time.js

```diff
@@ -170,17 +170,20 @@ class InputTime extends LitElement {
 	render() {
 		initIntervals(this.timeInterval);
 		const input = html`
-			<span class="${this.label && !this.labelHidden ? 'd2l-input-label' : 'd2l-offscreen'}" id="${this._dropdownId}-label">${this.label}</span>
+			<label
+				class="${this.label && !this.labelHidden ? 'd2l-input-label' : 'd2l-offscreen'}"
+				for="${this._dropdownId}-input"
+				id="${this._dropdownId}-label">${this.label}</label>
```

Comment:
Does it still need an `id` now that nothing is `aria-labelledby` pointing at it?

Reply:
It does, because the [dropdown menu](https://github.com/BrightspaceUI/core/blob/3667a91c1db69d6525eabef7a3566379da2ca745/components/inputs/input-time.js#L196) still needs to know what's labelling it - otherwise NVDA says '`undefined` menu' when identifying it.

Reply:
Ah I see that down there now -- thanks.